web/src/components/auth/SelfRegSnackbar.tsx

```diff
@@ -46,48 +47,57 @@ export const SelfRegSnackbar = (): ReactElement => {
     return <></>;
   }
 
+  const isSuccessAlert = alertMap[registrationAlertStatus] === "success";
+
+  if (isSuccessAlert) {
```

Comment:
This refactor looks really good. It's easy to read. One small suggestion, consider inlining line 50 into the conditional?